src/components/general/LeadCopyModal/styles.scss

```diff
@@ -19,7 +19,7 @@
             }
 
             h4 {
-                padding: var(--spacing-small) 0;
+                padding: var(--spacing-large) 0 0;
```

Comment:
Why change `LeadCopyModal` in the pull request related to private framework?

Reply:
Why not?